### ISSUE TITLE
Consider "thread pool size" over "thread pool depth"

### ISSUE THREAD

**Issue by binkley** [rufusraghunath/kelda#3]
https://github.com/rufusraghunath/kelda/blob/68687436769b458a6a65520f9f797b862fa51b6f/src/kelda/Kelda.ts#L10

The word "depth" many not be as clear as "size" in context.

**Comment by rufusraghunath** [comment]
It's the number of threads/Workers you're allowed to have at any one time. The queue only gets used to buffer Jobs when the `threadPoolDepth` is exceeded.

I chose "depth" over "size" because it evokes "pool" for me. Is "size" more standard in the Java world?

**Comment by binkley** [comment]
Yes, I think you'll find "size" is the more common word choice to describe this.
